DotNetCoreMasters/DotNetCoreMasters/Controllers/ItemsController.cs

```diff
@@ -8,16 +11,29 @@
 
 namespace DotNetCoreMasters.Controllers
 {
-    [ApiController]
-    [Route("[controller]")]
     public class ItemsController : Controller
     {
-        public IActionResult GetAll(int userId)
+        public int Get(int userId)
         {
             var res = new ItemService();
             var listOfStrings = res.GetAll(userId);
 
-            return Ok(listOfStrings);
+            return 0;
+        }
+
+        public void Post(ItemCreateBindingModel itemModel)
```

Comment:
Missing [FromBody] attribute. We need this since we removed [ApiController] attribute